Error processing line 1 of /home/mmk9369/.local/lib/python3.8/site-packages/vision-1.0.0-py3.8-nspkg.pth:

  Traceback (most recent call last):
    File "/share/apps/anaconda3/2020.07/lib/python3.8/site.py", line 169, in addpackage
      exec(line)
    File "<string>", line 1, in <module>
    File "<frozen importlib._bootstrap>", line 553, in module_from_spec
  AttributeError: 'NoneType' object has no attribute 'loader'

Remainder of file ignored
Error processing line 1 of /home/mmk9369/.local/lib/python3.8/site-packages/vision-1.0.0-py3.8-nspkg.pth:

  Traceback (most recent call last):
    File "/share/apps/anaconda3/2020.07/lib/python3.8/site.py", line 169, in addpackage
      exec(line)
    File "<string>", line 1, in <module>
    File "<frozen importlib._bootstrap>", line 553, in module_from_spec
  AttributeError: 'NoneType' object has no attribute 'loader'

Remainder of file ignored
Error processing line 1 of /home/mmk9369/.local/lib/python3.8/site-packages/vision-1.0.0-py3.8-nspkg.pth:

  Traceback (most recent call last):
    File "/share/apps/anaconda3/2020.07/lib/python3.8/site.py", line 169, in addpackage
      exec(line)
    File "<string>", line 1, in <module>
    File "<frozen importlib._bootstrap>", line 553, in module_from_spec
  AttributeError: 'NoneType' object has no attribute 'loader'

Remainder of file ignored
Error processing line 1 of /home/mmk9369/.local/lib/python3.8/site-packages/vision-1.0.0-py3.8-nspkg.pth:

  Traceback (most recent call last):
    File "/scratch/mmk9369/envs_dirs/deep/lib/python3.8/site.py", line 169, in addpackage
      exec(line)
    File "<string>", line 1, in <module>
    File "<frozen importlib._bootstrap>", line 553, in module_from_spec
  AttributeError: 'NoneType' object has no attribute 'loader'

Remainder of file ignored
Training with a single process on 1 GPUs.
Model cct_7_7x2_224_sine created, param count:4573735
Data processing configuration for current model + dataset:
	input_size: (3, 224, 224)
	interpolation: bicubic
	mean: (0.4353, 0.3773, 0.2872)
	std: (0.2966, 0.2455, 0.2698)
	crop_pct: 1.0
Using native Torch AMP. Training in mixed precision.
Scheduled epochs: 310
Traceback (most recent call last):
  File "train.py", line 855, in <module>
    main()
  File "train.py", line 613, in main
    train_metrics = train_one_epoch(
  File "train.py", line 686, in train_one_epoch
    output = model(input)
  File "/scratch/mmk9369/envs_dirs/deep/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1110, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/mmk9369/compact-transformers/src/cct.py", line 86, in forward
    return self.classifier(x)
  File "/scratch/mmk9369/envs_dirs/deep/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1110, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/mmk9369/compact-transformers/src/utils/transformers.py", line 199, in forward
    x += self.positional_emb
RuntimeError: The size of tensor a (4) must match the size of tensor b (196) at non-singleton dimension 1
